src/main/java/io/rcktapp/api/Api.java

```diff
@@ -30,9 +33,9 @@ public class Api extends Dto
    protected boolean          multiTenant = false;
    protected String           url         = null;
 
-   protected List<Db>         dbs         = new ArrayList();
-   protected List<Endpoint>   endpoints   = new ArrayList();
-   protected List<Action>     actions     = new ArrayList();
+   protected  List<Db>      dbs       = new ArrayList();
+   protected Set<Endpoint> endpoints = new TreeSet<>(Comparator.comparingInt(Rule::getOrder));
```

Comment:
Pretty simple all told to order these things... it doesn't work to just leave it without the comparator setting because the Rule Comparable doesn't seem to be an 'Endpoint' as far as its concerned. Perplexing.